src/addons/i2cdisplay.cpp

```diff
@@ -3,67 +3,69 @@
  * SPDX-FileCopyrightText: Copyright (c) 2021 Jason Skuby (mytechtoybox.com)
  */
 
-#include "modules/i2cdisplay.h"
+#include "addons/i2cdisplay.h"
 #include "enums.h"
 #include "helper.h"
-#include "storage.h"
+#include "storagemanager.h"
 #include "pico/stdlib.h"
 
-bool I2CDisplayModule::available() {
+bool I2CDisplayAddon::available() {
 	BoardOptions boardOptions = Storage::getInstance().getBoardOptions();
 	return boardOptions.hasI2CDisplay && boardOptions.i2cSDAPin != -1 && boardOptions.i2cSCLPin != -1;
 }
 
-void I2CDisplayModule::setup() {
-	BoardOptions options = Storage::getInstance().getBoardOptions();
+void I2CDisplayAddon::setup() {
+	BoardOptions boardOptions = Storage::getInstance().getBoardOptions();
 	obdI2CInit(&obd,
-	    options.displaySize,
-		options.displayI2CAddress,
-		options.displayFlip,
-		options.displayInvert,
+	    boardOptions.displaySize,
```

Comment:
May as well fix the indentation while we're here.
```suggestion
		boardOptions.displaySize,
```